src/main/java/edu/ucla/library/iiif/fester/verticles/V2ManifestVerticle.java

```diff
@@ -326,6 +326,12 @@ private void updateWork(final Message<JsonObject> aMessage) throws JsonProcessin
         final DeliveryOptions options = new DeliveryOptions();
         final JsonObject message = new JsonObject();
 
+        CsvParser.getMetadata(workRow, csvHeaders.getTitleIndex()).ifPresentOrElse(title -> {
```

Comment:
I had a question about this when Dawn mentioned she wanted _all_ metadata updated. Are we now updating all the metadata that we use in the manifest?

Reply:
I'm not sure, although I think it might be good to double check with Dawn that our implementation is updating the rest of the metadata as desired.

Reply:
(But I'd vote for that being separate from this ticket.)

Reply:
Maybe mention in standup today if she's there? I'm for us implementing what the ticket says, but perhaps surface the question in case she had something different in mind than what she wrote in the ticket.

Reply:
:+1: will do